src/purging_rmw.c

```diff
@@ -165,6 +165,70 @@ rmdir_recursive (char *path, short unsigned level)
   return status;
 }
 
+int
```

Comment:
```suggestion
static time_t
```

Reply:
Because you're returning a value of type time_t, may as well declare the function with that as well. And it can be static since this function is only going to be used in this file scope.

Reply:
Good catch